tests/php/test-class-monitor-css-transient-caching.php

```diff
@@ -5,6 +5,7 @@
  * @package AmpProject\AmpWP
  */
 
```

Comment:
This test should be moved to the `AmpProject\AmpWP\Tests` namespace. Suggested patch:

```diff
diff --git a/tests/php/test-class-monitor-css-transient-caching.php b/tests/php/src/BackgroundTask/MonitorCssTransientCachingTest.php
rename from tests/php/test-class-monitor-css-transient-caching.php
rename to tests/php/src/BackgroundTask/MonitorCssTransientCachingTest.php
--- a/tests/php/test-class-monitor-css-transient-caching.php	(revision cd4dd777102b5f30b32728e0c40cfaecc93b4097)
+++ b/tests/php/src/BackgroundTask/MonitorCssTransientCachingTest.php	(date 1606295939069)
@@ -1,20 +1,18 @@
 <?php
 /**
- * Class Test_Monitor_CSS_Transient_Caching.
- *
- * @package AmpProject\AmpWP
+ * Tests for the MonitorCssTransientCaching class.
  */
 
+namespace AmpProject\AmpWP\Tests\BackgroundTask;
+
+use AMP_Options_Manager;
 use AmpProject\AmpWP\BackgroundTask\BackgroundTaskDeactivator;
 use AmpProject\AmpWP\BackgroundTask\MonitorCssTransientCaching;
 use AmpProject\AmpWP\Option;
+use WP_UnitTestCase;
 
-/**
- * Test MonitorCssTransientCaching.
- *
- * @coversDefaultClass \AmpProject\AmpWP\BackgroundTask\MonitorCssTransientCaching
- */
-class Test_Monitor_CSS_Transient_Caching extends WP_UnitTestCase {
+/** @coversDefaultClass \AmpProject\AmpWP\BackgroundTask\MonitorCssTransientCaching */
+class MonitorCssTransientCachingTest extends WP_UnitTestCase {
 
 	/**
 	 * Whether external object cache is being used.
```

Reply:
Done in 3d07f78d6

Reply:
Only thing left to do here is to rename the file to `tests/php/src/BackgroundTask/MonitorCssTransientCachingTest.php`.

Reply:
Also, a small patch to be applied:

```diff
diff --git a/tests/php/test-class-monitor-css-transient-caching.php b/tests/php/test-class-monitor-css-transient-caching.php
--- a/tests/php/test-class-monitor-css-transient-caching.php	(revision 7cdde967dd63af7b7b1d8f945efe1a1060d55662)
+++ b/tests/php/test-class-monitor-css-transient-caching.php	(date 1608718679528)
@@ -45,7 +45,7 @@
 	 * Test whether an event is actually scheduled when the monitor is registered.
 	 *
 	 * @uses \AmpProject\AmpWP\BackgroundTask\CronBasedBackgroundTask::schedule_event
-	 * @uses \AmpProject\AmpWP\BackgroundTask\CronBasedBackgroundTask::deactivate
+	 * @uses \AmpProject\AmpWP\BackgroundTask\BackgroundTaskDeactivator::deactivate
 	 */
 	public function test_event_gets_scheduled_and_unscheduled() {
 		wp_set_current_user( self::factory()->user->create( [ 'role' => 'administrator' ] ) );
```

Reply:
Done in 1a4043156